react/intl-polyfill.ts

```diff
@@ -0,0 +1,33 @@
+import { path } from 'ramda'
+
+const locale = path<string>(['__RUNTIME__', 'culture', 'locale'], window)
+const [lang] = locale ? locale.split('-') : ''
+
+if (lang && window.Intl && !window.Intl.PluralRules) {
+  if (!window.IntlPluralRules || !window.IntlPluralRules.PluralRules) {
+    import('@formatjs/intl-pluralrules/polyfill' as any).then(() => {
+      import('@formatjs/intl-pluralrules/dist/locale-data/' + lang).then(() => {
+        window.Intl.PluralRules = window.IntlPluralRules.PluralRules
```

Comment:
There is no `window.IntlPluralRules` when you use `polyfill`. It already put itself directly into `window.Intl.PluralRules`

Not sure why you are not getting some error here. 🤔 
Did you check the console? It would log an unhandled promise exception.